client/me/security-2fa-disable/index.jsx

```diff
@@ -64,7 +64,7 @@ class Security2faDisable extends Component {
 				<div>
 					<p>
 						{ translate(
-							"You've enabled Two-Step Authentication. " +
+							"You've enabled two-step authentication. " +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'You've enabled Two-Step Authentication. While enabled, logging in to WordPress.com requires you to enter a unique passcode, sent via text message, in addition to your username and password.' )` *ES Score: 10*